schemas/scholarlyArticle.schema.tpl.json

```diff
@@ -0,0 +1,20 @@
+{
+  "_type": "https://openminds.ebrains.eu/publications/ScholarlyArticle",
+  "_extends": "creativeWork.schema.tpl.json",
+  "required": [
+      "isPartOf", "fullName", "releaseDate", "author"
+  ],
+  "properties": {
+    "isPartOf": {
+        "_instruction": "Indicates a CreativeWork that this item is part of.",
+        "_embeddedTypes": [
+          "https://openminds.ebrains.eu/publications/PublicationIssue",
+          "https://openminds.ebrains.eu/publications/PublicationVolume"
+        ]
+    },
```

Comment:
but embedding them does not make sense to me because multiple articles are part of the same issue or same volume. so they need to be independently registered as linked types so that multiple article can use them